app/Actions/SetupLamboStoreConfigs.php

```diff
@@ -0,0 +1,146 @@
+<?php
+
+namespace App\Actions;
+
+use LogicException;
+use App\Support\BaseAction;
+use Illuminate\Support\Facades\File;
+
+class SetupLamboStoreConfigs extends BaseAction
+{
+    /**
+     * Sets the configurations needed in Lambo Store.
+     *
+     * @return void
+     */
+    public function __invoke(): void
+    {
+        $this->projectName();
+
+        try {
+            $this->installPath();
+        } catch (LogicException $exception) {
+            $this->console->error($exception->getMessage());
+            exit(1);
+        }
+
+        $this->projectPath();
+
+        $this->projectUrl();
+
+        $this->dbName();
+
+        $this->checkStore();
+
+        if ($manualTest = false) {
```

Comment:
What's this?

Reply:
OK, this is as far as I made it on this code review. Gotta run; will come back to this later.